plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -53,17 +54,33 @@ func calculateScore(requested int64, capacity int64, node string) int64 {
 	return ((capacity - requested) * 10) / capacity
 }
 
-// Calculate the resource occupancy on a node.  'node' has information about the resources on the node.
+// The used capacity is calculated on a scale of 0-10
+// 0 being the lowest priority and 10 being the highest.
+// The more resources are used the higher the score is.
+func calculateUsedScore(requested int64, capacity int64, node string) int64 {
```

Comment:
I don't understand. This function appears to be exactly identical, character for character, to calculateUnusedScore(), except for the function name.. Am I missing something? I don't understand why you needed this change.